app/workers/scheduler/translation_service_scheduler.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Scheduler::TranslationServiceScheduler
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0
+
+  def perform
+    return unless TranslationService.configured?
+
+    Rails.cache.write('translation_service/languages', TranslationService.configured.languages)
```

Comment:
Our configuration expires cached values after 10 minutes by default, so this will not work as expected.